src/debugAdapter/debugNetwork.ts

```diff
@@ -97,4 +91,59 @@ export class DebugNetwork {
 
     return path.join(this.workingDirectory, directory);
   }
+
+  /**
+   * Gets all `LocalProject` from the `ItemType.LOCAL_SERVICE`.
+   *
+   * @returns an Array of `LocalProject` from the _Networks_ view.
+   */
+  private getProjects(): LocalProject[] {
```

Comment:
Should we try to unify `getProjects`+`getHost` and `selectGanachePortAndOptions`? It looks like they do almost the same. For instance, the icon in the quick pick is different, which is great, since it provides more context to the user.

Just mentioning it here so it doesn't fall through the cracks.

Reply:
Yes, I agree. We can have Helper to provide this information. We need to rethink that because there are a bunch of places calling the same information but in different ways...